inc/sccm.class.php

```diff
@@ -428,55 +441,53 @@ function getMedias($deviceid, $limit = 99999999) {
       return $data;
    }
 
-   static function install() {
-      $cronCollect = new CronTask;
+   static function install()
+   {
+      $cronCollect = $cronPush = new CronTask;
+      // Delete SCCMPush if existed
+     if ($cronPush->getFromDBbyName(__CLASS__, 'SCCMPush')) {
+         $cronPush->delete($cronPush->fields);
+      }
 
       if ($cronCollect->getFromDBbyName(__CLASS__, 'sccm')) {
 
          $cronCollect->fields["name"] = "SCCMCollect";
          $cronCollect->fields["hourmin"] = 4;
          $cronCollect->fields["hourmax"] = 5;
          $cronCollect->update($cronCollect->fields);
-
       } else if (!$cronCollect->getFromDBbyName(__CLASS__, 'SCCMCollect')) {
 
-         CronTask::register(__CLASS__, 'SCCMCollect', 7 * DAY_TIMESTAMP,
-            ['param' => 24, 'mode' => CronTask::MODE_EXTERNAL, 'hourmin' => 4, 'hourmax' => 5]);
-
+         CronTask::register(
+            __CLASS__,
+            'SCCMCollect',
+            7 * DAY_TIMESTAMP,
+            ['param' => 24, 'mode' => CronTask::MODE_EXTERNAL, 'hourmin' => 4, 'hourmax' => 5]
+         );
       }
-
-      CronTask::register(__CLASS__, 'SCCMPush', 7 * DAY_TIMESTAMP,
-            ['param' => 24, 'mode' => CronTask::MODE_EXTERNAL, 'hourmin' => 6, 'hourmax' => 7]);
    }
 
-   static function uninstall() {
+   static function uninstall()
+   {
       CronTask::unregister(__CLASS__);
    }
 
-   static function cronSCCMCollect($task) {
+   static function cronSCCMCollect($task)
+   {
       return self::executeCollect($task);
    }
 
-   static function cronSCCMPush($task) {
-      return self::executePush($task);
-   }
-
-   static function cronInfo($name) {
+   static function cronInfo($name)
+   {
       if ($name == "SCCMCollect") {
          return ['description' => __("Interface - SCCMCollect", "sccm")];
       }
-      if ($name == "SCCMPush") {
-         return ['description' => __("Interface - SCCMPush", "sccm")];
-      }
-
    }
 
-   static function executeCollect($task) {
+   static function executeCollect($task)
+   {
       ini_set('max_execution_time', 0);
```

Comment:
```suggestion
      ini_set('max_execution_time', 0);
      ini_set('memory_limit', '-1');
```

To prevent 

![image](https://github.com/pluginsGLPI/sccm/assets/7335054/e19ae025-4c37-43ac-9b13-0e572c65c50e)